style.css

```diff
@@ -0,0 +1,186 @@
+/* header css */
+.navigation {
+  border: none;
+  padding: 1rem 5rem;
+  position: sticky;
+  z-index: 9;
+  left: 0;
+  top: 0;
+  background-color: #fcfcfc;
+}
+
+.nav-left {
+  display: flex;
+}
+
+.nav-left span {
+  font-size: 0.8rem;
+}
+
+.btn {
+  padding: 0 8px;
+  height: 2.5rem;
+  margin: 5px 25px;
+}
+
+.nav-center input {
+  width: 350px;
+}
+
+.nav-right a {
+  color: black;
+}
+
+/* carousel slider css */
+#slider {
+  overflow: hidden;
+}
+
+#slider figure {
+  position: relative;
+  width: 500%;
+  margin: 0;
+  left: 0;
+  animation: 20s slider infinite;
+}
+
+#slider figure img {
```

Comment:
Maybe avoid id selectors for styling.

Reply:
Yes, I agree, but actually for the slider animation purpose, I used it.